src/cli/index.ts

```diff
@@ -49,7 +48,7 @@ async function sign(artifactPath: string) {
   console.error(`Created entry at index ${logIndex}, available at`);
   console.error(`${url}?logIndex=${logIndex}`);
 
-  console.log(JSON.stringify(Bundle.toJSON(bundle)));
+  console.log(JSON.stringify(bundle));
```

Comment:
No need for consumers to use `Bundle.toJSON` function as bundle is already in it's serialized form.